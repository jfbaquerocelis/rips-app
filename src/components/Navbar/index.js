import React from 'react';
import { Menu } from 'antd';

const Navbar = () => (
  <>
    <div className="logo" />
    <Menu theme="dark" mode="horizontal" defaultSelectedKeys={['2']}>
      <Menu.Item key="1">nav 1</Menu.Item>
      <Menu.Item key="2">nav 2</Menu.Item>
      <Menu.Item key="3">nav 3</Menu.Item>
    </Menu>
  </>
);

export default Navbar;
