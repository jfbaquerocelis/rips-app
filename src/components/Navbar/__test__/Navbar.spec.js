import React from 'react';
import { shallow } from 'enzyme';
import Navbar from '../index';

describe('Navbar component', () => {
  it('should render a component', () => {
    const wrapper = shallow(React.createElement(Navbar));

    expect(wrapper).toMatchSnapshot();
  });
});
