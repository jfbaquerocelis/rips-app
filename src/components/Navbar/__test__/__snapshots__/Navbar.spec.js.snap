// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component should render a component 1`] = `""`;
